src/daemon/utils_cache_mock.c

```diff
@@ -24,9 +24,10 @@
  *   Florian octo Forster <octo at collectd.org>
  */
 
-#include <errno.h>
 #include "utils_cache.h"
 
+#include <errno.h>
```

Comment:
unintended change?

Reply:
More like a drive-by fix. `"utils_cache.h"` should be included first.